### PR TITLE
enable tide merge automation for openshift/kube-storage-version-migrator

### PR DESCRIPTION
**Pull request by sanchezl**
New repo in support of [storage-migration-for-etcd-encryption](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/storage-migration-for-etcd-encryption.md).

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -194,6 +194,7 @@ tide:
     - openshift/kubernetes
     - openshift/kube-rbac-proxy
     - openshift/kube-state-metrics
+    - openshift/kube-storage-version-migrator
```
Comment (stevekuznetsov) [inline core-services/prow/02_config/_config.yaml]: Inclusion in this stanza only applies for `release-foo` branches -- do you want to also do `master`? Add to the other list in here, then, too

Comment (sanchezl) [reply core-services/prow/02_config/_config.yaml]: I've updated to also include in a section that excludes the `release-fooo` branches.

**Pull request merged**